br/README.md

```diff
@@ -1,131 +1,148 @@
-mc config host add minio $S3_ENDPOINT $MINIO_ACCESS_KEY $MINIO_SECRET_KEY && \
-mc mb minio/mybucket
+### TiKV-BR Command Line Description
+A br command consists of sub-commands, options, and parameters.
 
-# Backup to S3 compatible storage.
-bin/br backup full --pd pd0:2379 --storage "s3://mybucket/full" \
-    --s3.endpoint="$S3_ENDPOINT"
+- Sub-command: the characters without - or --, including `backup`, `restore`, `raw` and `help`.
+- Option: the characters that start with - or --.
+- Parameter: the characters that immediately follow behind and are passed to the sub-command or the option.
+#### Backup Raw Data
+To back up the cluster raw data, use the `tikv-br backup raw` command. To get help on this command, execute `tikv-br backup raw -h` or `tikv-br backup raw --help`.
+For example, backup raw data in TiKV cluster to `/tmp/backup` directory.
 
-# Drop database and restore!
-mysql -uroot -htidb -P4000 -E -e "DROP DATABASE test; SHOW DATABASES;" && \
-bin/br restore full --pd pd0:2379 --storage "s3://mybucket/full" \
-    --s3.endpoint="$S3_ENDPOINT"
+```
+tikv-br backup raw \
+    --pd "&{PDIP}:2379" \
+    -s "s3://backup-data/2022-09-16/" \
+    --s3.endpoint "s3://backup-data/2022-09-16/" \
+    --dst-api-version v2 \
+    --log-file="/tmp/br_backup.log
+```
+Explanations for some options in the above command are as follows:: 
+- `backup`: Sub-command of `tikv-br`.
+- `raw`: Sub-command of `backup`.
+- `-s` or `--storage`: Storage of backup files.
+- `"s3://backup-data/2022-09-16/"`: Parameter of `-s`, save the backup files in `"s3://backup-data/2022-09-16/"`.
+- `--pd`: Service address of `PD`.
+- `"${PDIP}:2379"`:  Parameter of `--pd`.
+- `--dst-api-version`: The `api-version`, 请见 [tikv-server config](https://docs.pingcap.com/zh/tidb/stable/tikv-configuration-file#api-version-%E4%BB%8E-v610-%E7%89%88%E6%9C%AC%E5%BC%80%E5%A7%8B%E5%BC%95%E5%85%A5) of backup files.
+- `v2`: Parameter of `--dst-api-version`, the optionals are `v1`, `v1ttl`, `v2`(Case insensitive). If no `dst-api-version` is specified, the `api-version` is the same with TiKV cluster of `--pd`.
+A progress bar is displayed in the terminal during the backup. When the progress bar advances to 100%, the backup is complete. The progress bar is displayed as follows:
+```
+br backup raw \
+    --pd "${PDIP}:2379" \
+    --storage "s3://backup-data/2022-09-16/" \
+    --log-file backupfull.log
+Backup Raw <---------/................................................> 17.12%.
 ```
 
-## Quick Start(tiup)
-
-```sh
-# Using tiup to start a TiDB cluster
-tiup playground --db 2 --pd 3 --kv 3 --monitor
-
-# Using tiup bench to generater test data.
-tiup bench tpcc --warehouses 1 prepare
-
-# How many row do we get? 300242 rows.
-mysql --host 127.0.0.1 --port 4000 -E -e "SELECT COUNT(*) FROM test.order_line" -u root -p
+TiKV-BR can do checksum between TiKV cluster and backup files after backup finish with the config  `--checksum=true`.  
```

Comment:
Suggest to introduce how to verify backup & restore in a separated section. I think users will much care about the correctness.
Can also talk about the checksum function of `client-go`.

Reply:
Added one section "Data Verification of Backup&Restore". PTAL~